tests/Smoke/StdinTest.php

```diff
@@ -56,7 +56,7 @@ public function testFixingStdin(): void
         $fileResult = str_replace("\n--- ".$path."\n", "\n--- php://stdin\n", $fileResult);
         $fileResult = str_replace("\n+++ ".$path."\n", "\n+++ php://stdin\n", $fileResult);
 
-        $path = str_replace('/', \DIRECTORY_SEPARATOR, basename(realpath($cwd)).'/'.$inputFile);
+        $path = str_replace('/', \DIRECTORY_SEPARATOR, $inputFile);
```

Comment:
@keradus 

In the previous PR you said
> isn't this change braking the usage of fixing the relatives and symlink/linked files?
>
> maybe it's worth to add a test cases like those

I'm not sure I understand what you mean. This is a test and the change adjust what we expect from stdout.